### PR TITLE
Throttle plugin support

### PR DESCRIPTION
**Pull request by velothump**
| Q               | A
| --------------- | ---
| New feature?    |yes
| License         | Apache 2.0

### What's in this PR?

Adds throttle plugin support

### Why?
Allows users to limit the rate of event outputs per Flow.

### Checklist

- [x] Code meets the [Developer Guide](https://github.com/banzaicloud/developer-guide)
- [x] User guide and development docs updated (if needed)
- [x] Related Helm chart(s) updated (if needed)

### CODE REVIEW

File: pkg/sdk/api/v1beta1/logging_types.go [hunk 1]
```diff
@@ -110,7 +110,7 @@ func (l *Logging) SetDefaults() (*Logging, error) {
 			copy.Spec.FluentdSpec.Image.Repository = "banzaicloud/fluentd"
 		}
 		if copy.Spec.FluentdSpec.Image.Tag == "" {
-			copy.Spec.FluentdSpec.Image.Tag = "v1.10.4-alpine-1"
+			copy.Spec.FluentdSpec.Image.Tag = "v1.10.5-alpine-1"
```
Comment (ahma) [inline pkg/sdk/api/v1beta1/logging_types.go:113]:
```suggestion
			copy.Spec.FluentdSpec.Image.Tag = "v1.10.4-alpine-2"
```

File: fluentd-image/v1.10/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 FROM alpine:3.11
-LABEL Description="Fluentd docker image" Vendor="Banzai Cloud" Version="1.10.4"
+LABEL Description="Fluentd docker image" Vendor="Banzai Cloud" Version="1.10.5"
```
Comment (ahma) [inline fluentd-image/v1.10/Dockerfile:2]:
```suggestion
LABEL Description="Fluentd docker image" Vendor="Banzai Cloud" Version="1.10.4"
```
This is still install fluentd version 1.10.4

**Comment by velothump** [comment]
Thank @ahma, updated now with your suggestions.

**Comment by ahma** [comment]
> Thank @ahma, updated now with your suggestions.

Thanks!

**Pull request merged**